Spanner/src/V1/Gapic/SpannerGapicClient.php

```diff
@@ -633,16 +633,13 @@ public function commit($session, $mutations, array $optionalArgs = [])
      *
      * @throws ApiException if the remote call fails
      */
-    public function createSession($database, array $optionalArgs = [])
+    public function createSession($database, $session, array $optionalArgs = [])
```

Comment:
Moving this from optional to required breaks BC and we'll need to fix it before merging!

See https://github.com/googleapis/gapic-generator-php/pull/459